src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -244,6 +249,12 @@ def _update_progress_bars(self):
             self.finish()
             return
 
+        if self.volclient.cloner.disable_clone_progress_bars:
+            log.info('mgr/vol/disable_clone_progress_bars is true, clone '
+                     'progress bars won\'t be printed.')
```

Comment:
Suggest "will not", quotes/apostrophes can confound parsing, something experience daily ;)